tests/App/FieldDescription/FieldDescriptionFactory.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Tests\App\FieldDescription;
+
+use Sonata\AdminBundle\Admin\AdminInterface;
+use Sonata\AdminBundle\Admin\FieldDescriptionInterface;
+use Sonata\AdminBundle\FieldDescription\FieldDescriptionFactoryInterface;
+use Sonata\AdminBundle\Tests\App\Admin\FieldDescription;
+
+final class FieldDescriptionFactory implements FieldDescriptionFactoryInterface
+{
+    public function create(AdminInterface $admin, string $name, array $options = []): FieldDescriptionInterface
```

Comment:
$admin is not used, is this correct?

Reply:
I would add `$fieldDescription->setAdmin($this);` here.

Reply:
> $admin is not used, is this correct?

I'll change it, the thing is that at first I thought about passing the admin, so the factory calls `$fieldDescription->setAdmin($admin)`, but then I thought it's better to call `$fieldDescription->setAdmin($this)` from the admin to avoid duplication and I forgot to change it.